docs/pages/database-access/reference/cli.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Database Access CLI Reference
+description: CLI reference for Teleport Database Access
+---
+
+# Database Access CLI Reference
+
+## tctl auth sign
+
+When invoked with a `--format=db` flag, produces a certificate authority, a
+client certificate and a private key file used for configuring Database
+Access with self-hosted database instances.
+
+```shell
+$ tctl auth sign --format=db --host=db.example.com --out=server --ttl=8760h
+```
+
+| Flag | Description |
+| - | - |
+| `--format` | When given value `db`, produces secrets in database compatible format. |
+| `--host` | Server name to encode in the certificate. Must match the hostname Teleport will be connecting to the database at. |
+| `--out` | Name prefix for output files. |
+| `--ttl` | Certificate validity period. |
+
+## tctl db ls
+
+Admin tool to list all databases registered with the cluster.
+
+```shell
+$ tctl db ls
+$ tctl db ls --format=yaml
+```
+
+| Flag | Description |
+| - | - |
+| `--format` | Output format, one of `text`, `yaml` or `json`. Defaults to `text`. |
+
+## tsh db ls
+
+Lists available databases and their connection information.
+
+```shell
+$ tsh db ls
+```
+
+Displays only the databases a user has access to (see [RBAC](./rbac.mdx)).
```

Comment:
Will want to verify the links for any files that have changed names or spots from the refactor:

```
Link to unknown file: `rbac.mdx`  missing-file  remark-validate-links
```